lang/usenglish/us_ffeatures.c

```diff
@@ -202,6 +246,261 @@ const cst_val *cg_content_words_in_phrase(const cst_item *p)
 	return float_val(ffeature_float(p,"R:SylStructure.parent.parent.R:Word.content_words_in") + ffeature_float(p,"R:SylStructure.parent.parent.R:Word.content_words_out")) ;//- (strcmp(ffeature_string(p,"R:SylStructure.parent.parent.R:Word.gpos"),"content")==0?1:0));
 }
 
+/* HTS features
+
+  HTS voices use a model with more contextual features, such as the total
+  number of words in the sentence, or the total number of syllables. The
+  following functions extract those features for US English texts.
+ */
+
+int item_after_length(const cst_item *n)
+{
+    int i = 0;
+    if (n == NULL)
+        return 0;
+    for (; n; n = n->n, i++);
```

Comment:
This is mainly my preference but I would prefer more verbosity in these for loops
```c
for (; n; n = n->n, i++)
{
    ; // do nothing
}
```